src/main/java/core/basesyntax/service/DataConverter.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface DataConverter {
+    List<FruitTransaction> converteData(List<String> dataFromFile);
```

Comment:
There seems to be a typo in the method name `converteData`. According to the Java naming conventions, method names should be verbs and correctly spelled. Consider renaming it to `convertData`.